tasks/au/data.py

```diff
@@ -46,15 +45,21 @@ class DownloadData(DownloadUnzipTask):
     state = Parameter()
     header = Parameter()
 
+    def version(self):
+        return 1
+
+    def requires(self):
+        return RepoFile(resource_id=self.task_id,
```

Comment:
Minor: there are 31 creations of `RepoFile` that all have `resource_id` and `version` taken from the Luigi task. What about creating a `repofile_from_task` factory method? Feel free to call me _javero_ 😅

Reply:
I considered doing something like that but I haven't done it because the current structure is our first approach and I would expect us to refine it a bit. I would wait until we have the structure completely clear, but we can discuss it if you think that it won't change (or at least won't change _a lot_)

Reply:
You're closer to the code, so up to you. I think that a factory function with named parameters and defaults should resits changes nicely, but whatever you decide will be fine.